book/02-git-basics/sections/aliases.asc

```diff
@@ -1,70 +1,70 @@
 [[_git_aliases]]
-=== Git Aliases
+=== Git Takma Adları
 
-(((aliases)))
-Before we finish this chapter on basic Git, there's just one little tip that can make your Git experience simpler, easier, and more familiar: aliases.
-We won't refer to them or assume you've used them later in the book, but you should probably know how to use them.
+(((takma adlar)))
+Bu bölümde temel Git'i bitirmeden önce, Git deneyiminizi daha kolay ve daha tanıdık hale getirecek küçük bir ipucu var: takma adlar.
+Onlara atıfta bulunmayacağız daha sonra kitapta kullandığınızı varsayıyoruz, ancak bunların nasıl kullanılacağını biliyor olmalısınız.
 
-Git doesn't automatically infer your command if you type it in partially.
-If you don't want to type the entire text of each of the Git commands, you can easily set up an alias for each command using `git config`.(((git commands, config)))
-Here are a couple of examples you may want to set up:
+Kısmen yazarsanız Git, komutunuzu otomatik olarak yorumlamaz.
+Git komutlarının her birinin tüm metnini yazmak istemiyorsanız, `git config`'yi kullanarak her komut için ayrı bir takma ad ayarlayabilirsiniz. (((Git komutları, yapılandırma)))
+Aşağıda, kurmak isteyebileceğiniz birkaç örnek verilmiştir:
 
-[source,console]
+[kaynak, konsol]
```

Comment:
Please don't translate the lines in the form `[...]`. These are asciidoc formatting commands. Please read about asciidoc to better understand what needs to be translated.

http://asciidoctor.org/docs/user-manual/